### PR TITLE
zh-translation:/faq/distributed-tracing/istio-copy-headers.md

### PR DESCRIPTION
**Pull request by lilinji**
https://github.com/servicemesher/istio-official-translation/issues/1395

### CODE REVIEW

**Comment by lilinji** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fistio%2Fistio.io%2Fpull%2F5336) for more info**.
@googlebot I signed it!

**Comment by lilinji** [comment]
> /ok-to-test

File: content/zh/faq/distributed-tracing/istio-copy-headers.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
 ---
-title: Why can't Istio propagate headers instead of the application?
+title: 为什么 Istio 只能传播标头文件而不能代替应用程序?{#why-can't-istio-propagate-headers-instead-of-the-application?}
```
Comment (rootsongjc) [inline content/zh/faq/distributed-tracing/istio-copy-headers.md]: 标题翻译有问题，请使用中文问号，不要在这个标题里增加英文锚点，只需要在正文里加即可。

File: content/zh/faq/distributed-tracing/istio-copy-headers.md [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
 weight: 20
 ---
 
-Although an Istio sidecar will process both inbound and outbound requests for an associated application instance, it has no implicit way of correlating
-the outbound requests to the inbound request that caused them. The only way this correlation can be achieved is if the application
-propagates relevant information (i.e. headers) from the inbound request to the outbound requests. Header propagation may be accomplished through client
-libraries or manually. Further discussion is provided in [What is required for distributed tracing with Istio?](/faq/distributed-tracing/#how-to-support-tracing).
+尽管 Istio Sidecar 将处理关联应用程序实例的入站和出站请求，它没有将出站请求与导致它们的入站请求相关联的隐式方法。可以实现这种关联的唯一方法是通过应用程序传播相关信息（i.e.标头）从入站请求到出站请求。头传播可以通过客户端库或手动完成。提供了进一步的讨论 [使用 Istio 进行分布式跟踪需要什么？](/faq/distributed-tracing/#how-to-support-tracing).
```
Comment (rootsongjc) [inline content/zh/faq/distributed-tracing/istio-copy-headers.md]: 链接前要加中文前缀 `/zh`。

File: content/zh/faq/distributed-tracing/istio-copy-headers.md [hunk 3]
```diff
@@ -1,9 +1,6 @@
 ---
-title: Why can't Istio propagate headers instead of the application?
+title: 为什么 Istio 只能传播标头文件而不能代替应用程序？
```
Comment (rootsongjc) [inline content/zh/faq/distributed-tracing/istio-copy-headers.md]: 为什么 Istio 不能代替应用程序传播标头？

File: content/zh/faq/distributed-tracing/istio-copy-headers.md [hunk 4]
```diff
@@ -1,9 +1,6 @@
...
 weight: 20
 ---
 
-Although an Istio sidecar will process both inbound and outbound requests for an associated application instance, it has no implicit way of correlating
-the outbound requests to the inbound request that caused them. The only way this correlation can be achieved is if the application
-propagates relevant information (i.e. headers) from the inbound request to the outbound requests. Header propagation may be accomplished through client
-libraries or manually. Further discussion is provided in [What is required for distributed tracing with Istio?](/faq/distributed-tracing/#how-to-support-tracing).
+尽管 Istio Sidecar 将处理关联应用程序实例的入站和出站请求，它没有将出站请求与导致它们的入站请求相关联的隐式方法。可以实现这种关联的唯一方法是通过应用程序传播相关信息（i.e.标头）从入站请求到出站请求。头传播可以通过客户端库或手动完成。提供了进一步的讨论 [使用 Istio 进行分布式跟踪需要什么？](/zh/faq/distributed-tracing/#how-to-support-tracing).
```
Comment (rootsongjc) [inline content/zh/faq/distributed-tracing/istio-copy-headers.md]: i.e. -> 例如

**Comment by lilinji** [comment]
> Need change.



> Need change.

**Pull request merged**